src/goto-instrument/nondet_static.cpp

```diff
@@ -46,6 +46,10 @@ bool is_nondet_initializable_static(
   if(ns.lookup(id).type.get_bool(ID_C_no_nondet_initialization))
     return false;
 
+  // is the type explicitly marked as not to be initialized?
+  if(ns.lookup(id).type.get_bool(ID_C_no_initialization_required))
+    return false;
```

Comment:
Ok, but I note that the same `ns.lookup(id)` now happening at least three times. I ... don't think that's very efficient.

Reply:
Done